src/Magnum/Math/Test/DualComplexTest.cpp

```diff
@@ -445,6 +449,24 @@ void DualComplexTest::transformPoint() {
     CORRADE_COMPARE(transformedB, Vector2(-2.918512f, 2.780698f));
 }
 
+void DualComplexTest::strictWeakOrdering()
+{
+    StrictWeakOrderingFunc o;
+    const DualComplex a{{1, 0}, {1, 3}};
+    const DualComplex b{{1, 2}, {3, 4}};
+    const DualComplex c{{1, 0}, {1, 4}};
+    const DualComplex d{{1, 0}, {1, 3}};
```

Comment:
For type clarity and consistency I'm adding explicit `.0f` to all literals in constructors of floating-point types, even though it's not strictly needed. Similarly elsewhere.

Also, probably just slipped through no newline before a `{` ;)